tests/std/tests/VSO_0157762_feature_test_macros/test.compile.pass.cpp

```diff
@@ -364,7 +364,7 @@ STATIC_ASSERT(__cpp_lib_byteswap == 202110L);
 #endif
 #endif
 
-#if _HAS_CXX20 && defined(__cpp_char8_t)
+#if defined(__cpp_char8_t)

```

Comment:
The special `env.lst` here needs to be updated in order to exercise this change.